python/openai/README.md

```diff
@@ -255,7 +246,7 @@ docker run -it --net=host --gpus all --rm \
   -v ${HOME}/.cache/huggingface:/root/.cache/huggingface \
   -e HF_TOKEN \
   -e TRTLLM_ORCHESTRATOR=1 \
-  nvcr.io/nvidia/tritonserver:24.08-trtllm-python-py3
+  nvcr.io/nvidia/tritonserver:24.11-trtllm-python-py3
```

Comment:
Was `24.11` accidental or the version should be 24.12?

Reply:
Intentional 24.11 because we don't have a 24.12 trtllm ready to test the changes yet. It can be bumped to 24.12 once we can confirm it works.

Reply:
Good to know that. Thanks!